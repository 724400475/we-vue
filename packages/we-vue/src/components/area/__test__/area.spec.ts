import { mount } from '@vue/test-utils'
import Area from '../area'
// import { slowVerticalDrag } from '@/test/unit/utils'

describe('area picker', () => {
  test('create', () => {
    const wrapper = mount(Area)

    expect(wrapper.name()).toBe('wv-area')

    expect(wrapper.html()).toMatchSnapshot()
  })
})
