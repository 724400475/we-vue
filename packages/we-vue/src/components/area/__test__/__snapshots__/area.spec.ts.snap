// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`area picker create 1`] = `
<div>
  <div class="weui-mask" style="display: none;"></div>
  <div class="weui-picker" style="display: none;">
    <div class="weui-picker__hd">
      <div class="weui-picker__action">取消</div>
      <div class="weui-picker__action">确定</div>
    </div>
    <div class="weui-picker__bd" style="height: 238px;">
      <div class="weui-picker__group">
        <div class="weui-picker__mask" style="background-size: 100% 102px;"></div>
        <div class="weui-picker__indicator" style="top: 102px;"></div>
        <div class="weui-picker__content" style="transform: translate3d(0, 102px, 0);"></div>
      </div>
      <div class="weui-picker__group">
        <div class="weui-picker__mask" style="background-size: 100% 102px;"></div>
        <div class="weui-picker__indicator" style="top: 102px;"></div>
        <div class="weui-picker__content" style="transform: translate3d(0, 102px, 0);"></div>
      </div>
      <div class="weui-picker__group">
        <div class="weui-picker__mask" style="background-size: 100% 102px;"></div>
        <div class="weui-picker__indicator" style="top: 102px;"></div>
        <div class="weui-picker__content" style="transform: translate3d(0, 102px, 0);"></div>
      </div>
    </div>
  </div>
</div>
`;
